vault/resource_namespace.go

```diff
@@ -77,7 +83,9 @@ func namespaceRead(d *schema.ResourceData, meta interface{}) error {
 		return nil
 	}
 
-	d.SetId(resp.Data["id"].(string))
+	d.SetId(resp.Data["path"].(string))
```

Comment:
Question! If people are already using this, and then they upgrade the Terraform Vault Provider, and Terraform detects that it needs to update the ID, will that result in Terraform outputting that it's detected that a namespace needs to be modified? Or I suppose in a broader sense, what will the migration path look like?

Just asking about backwards compat so I can factor that into whether this belongs in a major, minor, or patch release. Thank you!